package.json

```diff
@@ -32,10 +32,11 @@
     "blueimp-md5": "2.18.0",
     "commander": "6.2.0",
     "gl-matrix": "3.3.0",
-    "jszip": "3.5.0",
+    "jszip": "3.2.0",
     "pako": "1.0.11",
     "seedrandom": "3.0.5",
     "shelljs": "0.8.4",
+    "stream": "0.0.2",
```

Comment:
See if https://github.com/Kitware/vtk-js/pull/1651#issuecomment-741909256 allows us to remove this stream dependency.

Reply:
I tried this, and also tried with `readable-stream`, it doesn't seem to help.

Reply:
Note: I am testing with `npm run example -- SceneExplorer` because it uses the JSZipDataAccessHelper. `npm run build` and other examples do seem to work. No luck with that example though.

Reply:
Fixed it. I also had to change the example template.